src/modules/interactive.js

```diff
@@ -0,0 +1,13 @@
+import Function from './function.js';
+
+const Status = document.querySelector('#checkbox');
+if (Status.checked) {
+  this.completed = 'true';
+  this.saveTasksToLocalStorage();
+} else {
+  this.completed = 'false';
+}
+
+Function();
+
+export default Status;
```

Comment:
- [ ] Hi, Nice job so far. I would advise you to implement the function for the `Clear all completed` button using the `Array.filter()` method. Make it work and submit it again for approval. I am requesting this because it is of the requirements of this milestone.

- NB: This comment is not in line to the concern code because you did not make the file available to be seen.